### PR TITLE
Adds a comment as requested

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: node/core/bitfield-signing/src/lib.rs [hunk 1]
```diff
@@ -210,6 +210,10 @@ async fn construct_availability_bitfield(
 	let availability_cores = get_availability_cores(relay_parent, sender).await?;
 
 	// Wrap the sender in a Mutex to share it between the futures.
+	//
+	// We use a `Mutex` here to not `clone` the sender inide the future, because
```
Comment (montekki) [inline node/core/bitfield-signing/src/lib.rs:214]:
```suggestion
	// We use a `Mutex` here to not `clone` the sender inside the future, because
```

**Comment by pepyakin** [comment]
❤️

**Comment by pepyakin** [comment]
bot merge

**Comment by bkchr** [comment]
@pepyakin an approval would be nice xD

**Review by pepyakin — APPROVED** [review]

**Comment by pepyakin** [comment]
that's annoying though. I would treat

bot merge

as an implicit approve by me

**Comment by pepyakin** [comment]
bot merge

**Pull request merged**